assets/src/components/carousel/index.js

```diff
@@ -0,0 +1,259 @@
+/**
```

Comment:
Search input is a good idea, but we an defer that for the future.

Reply:
Yeah, the enlarged dot represents the center item, but the update I pushed this morning that removed the empty space at the start and end of the carousel made it a little weird. Maybe I will just remove the dot enlargement because it is confusing now.

Reply:
Removed the "current" dot style in 3af1d68ba. Trying to reproduce weird left/right scroll behavior now.

Reply:
@pierlon Could you re-test using the left/right arrows to scroll with the carousel? There are a few possibilities that would tie into browser compatibility, so if it's still not working right, let me know what browser you're using.